### ISSUE TITLE
Setting to allow swipe to scroll, instead of go through history

### ISSUE THREAD

**Issue by gruber** [holzschu/a-shell#753]
I often run commands that generate a page or two of output. I’d rather have swipe scroll, like a normal app, than have swipe go through my shell history. Is there a hidden setting for this?

**Comment by holzschu** [comment]
In theory, single-finger-swipe does scroll up and down, while two-fingers-swipe does go through history.

From your question, I infer that this is not the case on your device?

**Comment by gruber** [comment]
Yes. On my iPad (iPadOS 17.4), single-finger swiping goes through history. But on my iPhone, single-finger swiping still scrolls. I didn’t notice the difference because I use a-Shell on my iPad so much more often than on iPhone. Any ideas what’s wrong?

**Comment by holzschu** [comment]
I've had another report of the same issue. It seems it was fixed by terminating the iPad window with "exit".

**Comment by gruber** [comment]
I think that fixed it for me too. I didn’t type `exit` but did restart the iPad, which I think implies an exit. Now single-finger swiping scrolls again.
